### ISSUE TITLE
google.api_core.exceptions.InternalServerError: 500 Received RST_STREAM with error code 2

### ISSUE THREAD

**Issue by Rstar1998** [googleapis/python-datastream#137]
So I have the following code and when I execute in cloud shell ( I have all necessary permissions ) I get a weird error 
```
from google.cloud import datastream_v1


def sample_create_connection_profile():
    # Create a client
    client = datastream_v1.DatastreamClient()

    # Initialize request argument(s)
    connection_profile = datastream_v1.ConnectionProfile()
    
    connection_profile.mysql_profile.hostname = ""
    connection_profile.mysql_profile.username = ""
    connection_profile.mysql_profile.password = ""
    connection_profile.mysql_profile.port=3306
    connection_profile.forward_ssh_connectivity.hostname=""
    connection_profile.forward_ssh_connectivity.username=""
    connection_profile.forward_ssh_connectivity.port=22
    #connection_profile.forward_ssh_connectivity.password=""	
    connection_profile.forward_ssh_connectivity.private_key="""
    

    """
    #connection_profile.mysql_profile.database_service = "database_service_value"
    connection_profile.display_name = "display_name_value"

    request = datastream_v1.CreateConnectionProfileRequest(
        parent="parent_value",
        connection_profile_id="connection_profile_id_value",
        connection_profile=connection_profile,
    )

    # Make the request
    operation = client.create_connection_profile(request=request)

    print("Waiting for operation to complete...")

    response = operation.result()

    # Handle the response
    print(response)

sample_create_connection_profile()
# [END datastream_v1_generated_datastream_v1_CreateConnectionProfile_sync]
```
ERROR

```
$ python3 datastream.py 

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/google/api_core/grpc_helpers.py", line 72, in error_remapped_callable
    return callable_(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/grpc/_channel.py", line 946, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/usr/local/lib/python3.9/dist-packages/grpc/_channel.py", line 849, in _end_unary_response_blocking
    raise _InactiveRpcError(state)
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
        status = StatusCode.INTERNAL
        details = "Received RST_STREAM with error code 2"
        debug_error_string = "UNKNOWN:Error received from peer ipv4:74.125.68.95:443 {created_time:"2022-09-23T12:00:07.866550418+00:00", grpc_status:13, grpc_message:"Received RST_STREAM with error code 2"}"
>

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/name/datastream.py", line 42, in <module>
    sample_create_connection_profile()
  File "/home/name/datastream.py", line 33, in sample_create_connection_profile
    operation = client.create_connection_profile(request=request)
  File "/home/name/.local/lib/python3.9/site-packages/google/cloud/datastream_v1/services/datastream/client.py", line 879, in create_connection_profile
    response = rpc(
  File "/usr/local/lib/python3.9/dist-packages/google/api_core/gapic_v1/method.py", line 154, in __call__
    return wrapped_func(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/google/api_core/grpc_helpers.py", line 74, in error_remapped_callable
    raise exceptions.from_grpc_error(exc) from exc
google.api_core.exceptions.InternalServerError: 500 Received RST_STREAM with error code 2

```
Can anyone help me to identify exactly where I am wrong ?

**Comment by Rstar1998** [comment]
Any updates on this issue ? Have to created everything manually 😞

**Comment by parthea** [comment]
Closing this bug as obsolete. The issue was fixed  b/246509350. Please open a new issue if the problem re-appears.
